newIDE/app/src/AssetStore/index.js

```diff
@@ -115,7 +118,51 @@ export const AssetStore = React.forwardRef<Props, AssetStoreInterface>(
       searchText,
       setSearchText: setAssetStoreSearchText,
       clearAllFilters: clearAllAssetStoreFilters,
+      assetFiltersState,
+      assetShortHeadersById,
```

Comment:
Can you remove exposing this and instead add a `getAssetShortHeaderFromId: (id: string) => AssetShortHeader | null` to the state?
It will be clearer and easier to find usage of this without exposing internal "details" about the state.